src/note/forms.py

```diff
@@ -7,3 +7,13 @@ class NoteForm(forms.ModelForm):
     class Meta:
         model = Note
         fields = ['note', 'client', 'priority']
+
+        widgets = {
+            'note': forms.Textarea(attrs={'rows': 5}),
+            'client': forms.Select(
+                attrs={'class': 'ui status_to dropdown'}
```

Comment:
Semantic UI provides a searchable dropdown widget, which is more efficient when list has a lot of elements: http://semantic-ui.com/modules/dropdown.html#search-selection.

Also, what is the use of the class _status_to_ ?